Add-ons/Starter-Kit-TEMP/Content-Types/Global-Content/index.md

```diff
@@ -0,0 +1,64 @@
+---
+versionFrom: 8.0.0
+---
+
+# The Global Content
+		
+
+This Node lets you create the widgets, that you will then later be able to, use in The Global Content Widget.
+When you have created a widget here it will work as a template in the future, so that you will only have to set it up once, saving you a lot of time in the future. 
+		
+
+Setting your widgets up with this feature ensures that every time you make a new page, you can get the same design on each page.
+This is because you made this template, all you will need to do is add it in the Global Content Widget, making it super straight forward to keep the same design on all of your pages.
```

Comment:
Removed "This is because you made this template", as this is already implied by the first part of this section.

Otherwise it's a really good introduction to the Global Content feature.

```suggestion
All you will need to do is add them in the Global Content Widget, making it super straight forward to keep the same design on all of your pages.
```